lib/components/viewers/viewers.css

```diff
@@ -273,3 +273,59 @@ a.routeDetails {
   float: right;
   width: 50px;
 }
+
+/* Route Viewer Updates */
+.search-and-filter {
+  margin-top: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+}
+.search-and-filter select {
+  margin-left: 10px;
+  text-overflow: ellipsis;
+  min-width: 105px;
+
+  border: none;
+  background: #eee;
+  border-radius: 5px;
+  padding: 5px;
+}
+.search-and-filter select option {
+  /* This allows the dropdowns to shrink and stretch */
+  max-width: 0;
+}
+
+.search-and-filter .routeFilter {
+  display: grid;
+  align-items: center;
+  grid-template-columns: 0fr 2fr 1fr;
+  width: 100%;
+}
+.search-and-filter .routeSearch {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.search-and-filter .routeSearch input {
+  border: none;
+  padding: 0.125em 0.5em;
+  border-radius: 5px;
+  margin-left: 5px;
```

Comment:
Set to 10px.

Reply:
Good catch thanks! Fixed in 3bcaf1302c8766d59aa3eb92f837832bb8ce3eaa